src/gas/index.ts

```diff
@@ -231,6 +231,27 @@ function sell(coin: CoinName): string {
   });
 }
 
+function testEmail(): string {
```

Comment:
Not sure we need this test function in the build... I would not commit it.
Right now I have no infra to have test functions separately